src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowDelete.cs

```diff
@@ -74,6 +100,38 @@ public override DbCommand GetCommand(DbConnection connection)
             return command;
         }
 
+        /// <summary>
+        /// Runs a query using the where clause to determine if duplicates are found (causes issues when deleting).
+        /// </summary>
+        private bool checkWhereDuplicate(WhereClause where, string input, DbConnection connection)
+        {
+            bool verifyStatus = false;
+            using (DbCommand command = connection.CreateCommand())
+            {
+                command.CommandText = input;
+                command.Parameters.AddRange(where.Parameters.ToArray());
+                using (DbDataReader reader = command.ExecuteReader())
+                {
+                    try
+                    {
+                        while (reader.Read())
+                        {
+                            if (reader.GetInt32(0) != 1)
+                            {
+                                verifyStatus = true;
+                            }
+                        }
+                        reader.Close();
+                    }
+                    catch
+                    {
+                        //Likely means there was nothing found that matched the query.
```

Comment:
I don't think this statement is true - that should be a rare case and it's more expected that a network error or something would happen.

Regardless I'd just remove the comment - I think it's clear enough that we just want to default to false if we can't determine what it is from the query itself.